.github/workflows/lint-mixins.yml

```diff
@@ -0,0 +1,83 @@
+---
+name: Mixin
+
+permissions:
+  contents: read
+
+on:
+  # To conserve resources we only run tests against main in PRs
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  check-for-changed-mixins:
+    name: Check for changes
+    runs-on: ubuntu-latest
+    outputs:
+      changed-mixins: ${{ steps.changed-mixins.outputs.all_changed_files }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: 1.18
+      
+      - name: Install CI dependencies
+        run: make install-ci-deps
+      
+      - name: Get changed mixins
+        id: changed-mixins
+        uses: tj-actions/changed-files@v44
+        with:
+          dir_names: true
+          dir_names_exclude_current_dir: true
+          dir_names_max_depth: 1
+          files: | 
+            **-mixin/
```

Comment:
some observ-libs would have sample mixin.libsonnet file too.
Should we try to lint it as well?

For example: kafk-observ-lib/mixin.libsonnet